Makefile

```diff
@@ -27,10 +27,16 @@ test-all: test
 test:
 	$(MAKE) -C demo test
 	$(MAKE) -C x test
-	$(MAKE) -C tests/e2e/ test
 
-test-e2e:
+test-e2e: build-docker-e2e test-e2e-cache
+
+test-e2e-cache:
 	$(MAKE) -C tests/e2e/ test
```

Comment:
This is another set of e2e tests using `ibctesting`. We would still need to keep it apart from the newly added dockerised e2e tests

Reply:
Let's rename these tests then (if they are not void).